docs/terms/data_docs.md

```diff
@@ -1,14 +1,100 @@
+
+## Use cases
+
+<SetupHeader/>
+
+You can configure multiple Data Docs sites while setting up your Great Expectations project.  This allows you to tailor the information that is displayed by Data Docs as well as how they are hosted.  For more information on setting up your Data Docs, please reference our [guides on how to configure them for specific hosting environments](../guides/setup/index.md#data-docs).
+
+<CreateHeader/>
+
+You can view your saved Expectation Suites in Data Docs.  
+
+<ValidateHeader/>
+
+Saved Validation Results will be displayed in any Data Docs site that is configured to show them.  If you build your Data Docs from the Data Context, the process will render Data Docs for all of your Validation Results.  Alternatively, you can use the `UpdateDataDocsAction` Action in a Checkpoint's `action_list` to update your Data Docs with just the Validation Results generated by that checkpoint.
+
+## Features
+
+### Readability
+
+Data Docs provide a clean, human-readable way to view your Expectation Suites and Validation Results without you having to manually parse their stored values and configurations.  You can also [add comments to your Expectations that will be displayed in your Data Docs](../guides/expectations/advanced/how_to_add_comments_to_expectations_and_display_them_in_data_docs.md), if you feel they need further explanation.
+
+### Versatility
+
+There are multiple use cases for displaying information in your Data Docs.  Three common ones are:
+
+1. Visualize all Great Expectations artifacts from the local repository of a project as HTML: Expectation Suites,
+   Validation Results and profiling results.
+1. Maintain a "shared source of truth" for a team working on a data project. Such documentation renders all the
+   artifacts committed in the source control system (Expectation Suites and profiling results) and a continuously
+   updating data quality report, built from a chronological list of validations by run id.
+1. Share a spec of a dataset with a client or a partner. This is similar to API documentation in software development.
+   This documentation would include profiling results of the dataset to give the reader a quick way to grasp what the
+   data looks like, and one or more Expectation Suites that encode what is expected from the data to be considered
+   valid.
+
+To support these (and possibly other) use cases Great Expectations has a concept of "data documentation site". Multiple sites can be configured inside a project, each suitable for a particular data documentation use case.
+
```

Comment:
Should we have a comma after use-case?

Reply:
A comma could be included there, but isn't necessary.

Reply:
oh English why are you so tricky